### ISSUE TITLE
Module failed to run. Failed to resolve scenario. ( All Scenarios )

### ISSUE THREAD

**Issue by ZPayneee** [cliffe/SecGen#164]
I just started trying to get Sec Gen to work. Was having problems before getting the vm to boot up as it was timing out when trying to boot then giving an error and shutting down. I tried to update gem and then didnt even get that far and get this error with or without sudo.

EDIT: So I re ran the "bundler install" command and it fixed the Failed to resolve scenario errors I was getting which I think was due to the bundler file not being in the right place after updating it. I am however still getting the time out error when it is trying to connect to the VM. This is the error I get

Copying post-provision testing class
VM(s) can be built using 'vagrant up' in /home/student/SecGen/projects/SecGen20200131_121339
Project files created.
Building project: /home/student/SecGen/projects/SecGen20200131_121339
Loading vagrant (up) in /home/student/SecGen/projects/SecGen20200131_121339
Bringing machine 'escalation' up with 'virtualbox' provider...
==> escalation: Importing base box 'modules_bases_debian_stretch_server'...
==> escalation: Matching MAC address for NAT networking...
==> escalation: Setting the name of the VM: SecGen20200131_121339_escalation_1580501878204_85334
==> escalation: Clearing any previously set network interfaces...
==> escalation: Preparing network interfaces based on configuration...
    escalation: Adapter 1: nat
    escalation: Adapter 2: hostonly
==> escalation: Forwarding ports...
    escalation: 22 (guest) => 2222 (host) (adapter 1)
==> escalation: Running 'pre-boot' VM customizations...
==> escalation: Booting VM...
==> escalation: Waiting for machine to boot. This may take a few minutes...
    escalation: SSH address: 127.0.0.1:2222
    escalation: SSH username: vagrant
    escalation: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.
Non-zero exit status...
Error provisioning VMs, destroying VMs and exiting SecGen.
Loading vagrant (destroy -f) in /home/student/SecGen/projects/SecGen20200131_121339
==> escalation: Forcing shutdown of VM...
==> escalation: Destroying VM and associated drives...
student@ubuntu:~/SecGen$

**Comment by jeremytourville** [comment]
Please list your command line syntax when calling for the scenario to be run.  That will help me get a better sense of what's happening.  You shouldn't need to use sudo for anything if your directory structure is setup properly.  I am using Ubuntu to run SecGen.  I started with a minimal install.  From there I created ~/bin directory and then just followed the directions.  Hope this helps some.

**Comment by Binx-bit** [comment]
I am also having the same problem however, I was not able to remedy it by running the "bundle install" command. After all modules fail I get the following error:
Module failed to run (bundle exec ruby modules/generators/compression/zip/secgen_local/local.rb)
Failed to resolve scenario.
No conflicts, but failed to resolve scenario -- this is a sign there is something wrong in the config (scenario / modules)
Please review the scenario -- something is wrong.

The command used was the default CTF event : ruby secgen.rb --scenario scenarios/ctf/flawed_fortress_1.xml run

Any help would be awesome.

**Comment by NeodymiumPhish** [comment]
I'm having the same issue.  I've tried 5 scenarios (including the default), and I always get the "Timed out while waiting for the machine to boot. ..." 

The last time I did this, I built the metactf from the examples/ctf_challenge_examples folder, and I modified the network to "public_network" instead of "private_network".  I see that this change had an effect (Adapter 2: bridged) in the vagrant output, but it still sat at "metactf: SSH auth method: private key" for a few minutes before the red timed out text and vagrant destroy -f ran.  

Here's a screenshot of the latest entry, where I just ran `vagrant up` from the project folder (I switched the network type back to private):

![Screenshot from 2020-05-15 14-37-25](https://user-images.githubusercontent.com/7989828/82089657-9f108e80-96b9-11ea-8a58-4530bb0562da.png)
